gallery/src/components/NxDropdown/NxDropdownWithNxThreatIndicatorExample.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { NxDropdown, useToggle, NxThreatIndicator } from '@sonatype/react-shared-components';
+
+function NxDropdownWithNxThreatIndicatorExample() {
+  const [isOpen, onToggleCollapse] = useToggle(false),
+      onClick = () => { alert('click'); },
+      labelElement = <><NxThreatIndicator threatLevelCategory="moderate" /><span>Moderate</span></>;
+
+  return (
+    <NxDropdown label={labelElement}
+                className="extra-class"
```

Comment:
This seems extraneous to the purpose of the example.

Reply:
Which part? The dropdown example?

Reply:
> Which part? The dropdown example?

The `className`.  It's not doing anything or demonstrating anything, right?